packages/nakama-js/dist/api.gen.d.ts

```diff
@@ -310,27 +316,39 @@ export interface ApiWriteStorageObject {
 export interface ApiWriteStorageObjectsRequest {
     objects?: Array<ApiWriteStorageObject>;
 }
+export interface ProtobufAny {
+    type_url?: string;
+    value?: string;
+}
+export interface RpcStatus {
+    code?: number;
+    details?: Array<ProtobufAny>;
+    message?: string;
+}
 export declare const NakamaApi: (configuration?: ConfigurationParameters) => {
     doFetch(urlPath: string, method: string, queryParams: any, body?: any, options?: any): Promise<any>;
     healthcheck(options?: any): Promise<any>;
     getAccount(options?: any): Promise<ApiAccount>;
     updateAccount(body: ApiUpdateAccountRequest, options?: any): Promise<any>;
-    authenticateCustom(body: ApiAccountCustom, create?: boolean, username?: string, options?: any): Promise<ApiSession>;
-    authenticateDevice(body: ApiAccountDevice, create?: boolean, username?: string, options?: any): Promise<ApiSession>;
-    authenticateEmail(body: ApiAccountEmail, create?: boolean, username?: string, options?: any): Promise<ApiSession>;
-    authenticateFacebook(body: ApiAccountFacebook, create?: boolean, username?: string, sync?: boolean, options?: any): Promise<ApiSession>;
-    authenticateFacebookInstantGame(body: ApiAccountFacebookInstantGame, create?: boolean, username?: string, options?: any): Promise<ApiSession>;
-    authenticateGameCenter(body: ApiAccountGameCenter, create?: boolean, username?: string, options?: any): Promise<ApiSession>;
-    authenticateGoogle(body: ApiAccountGoogle, create?: boolean, username?: string, options?: any): Promise<ApiSession>;
-    authenticateSteam(body: ApiAccountSteam, create?: boolean, username?: string, options?: any): Promise<ApiSession>;
+    authenticateApple(body: ApiAccountApple, create?: boolean | undefined, username?: string | undefined, options?: any): Promise<ApiSession>;
```

Comment:
This `| undefined` was introduced via a change to the `tsconfig`.  I don't think it really matters since it's in the generated core, but just thought I'd note.